docs/nodes/build/run-avalanche-node-manually.md

```diff
@@ -108,19 +108,17 @@ Download the AvalancheGo repository into your `$GOPATH`:
 cd $GOPATH
 mkdir -p src/github.com/ava-labs
 cd src/github.com/ava-labs
-git clone git@github.com:ava-labs/avalanchego.git
+git clone https://github.com/ava-labs/avalanchego.git
```

Comment:
If we change this, we should make consistent in all docs to use https.

Reply:
I did check before opening the PR, and it does look like https is used everywhere in the docs (at least it will be after merging this PR).

Reply:
Actually, I've missed one place. Fixed the mistake now. HTTPS is used everywhere, and next to it there is an info box about SSH.